echidna/scripts/rebin.py

```diff
@@ -0,0 +1,48 @@
+""" Example smearing script
+
+This script:
+  * Reads in spectrum from hdf5
+  * Rebins spectrum with user input number of bins
+  * Saves rebinned spectrum to given output or, by default, to the same
+    directory with _rebin added to the filename. 
```

Comment:
Sorry I'm being a bit picky but could you either end all bullet points with a full-stop or none (unless the bullet has multiple sentences) not mix and match.